README.md

```diff
@@ -15,3 +15,23 @@ Theia is a cloud & desktop IDE implemented in TypeScript.
 ### Running the examples
 
 You can find more details on how to run the examples [here](doc/Developing.md).
+ ### Running the electron example
+ ```
+ git clone https://github.com/TypeFox/Theia.git \
+ && cd Theia \
+ && npm install \
+ && cd examples \
+ && npm run install:theia \
+ && npm install \
+ && npm run cold:start:electron
```

Comment:
These are redundant (and obsolete) steps. It is already available under the `Developing.md`.